dtfs-central-api/src/services/tfm/deal-cancellation.service.ts

```diff
@@ -77,6 +80,13 @@ export class DealCancellationService {
         dealType,
       });
 
+      await PortalDealService.addGefDealCancelledActivity({
```

Comment:
should any of these be in a try catch in this file since they are async calls

Reply:
None of the existing services have try/catch handlers - would like them to be aligned.

I imagine this decision was made so that errors bubble up.

@MarRobSoftwire ?

Reply:
Yep thats the idea, because we want errors thrown in the repositories/services to be handled at the controller level we dont want a try/catch here.

If we were going to add one we'd need to write

```
catch (error) {
  throw error
}
```
which es-lint prevents you from doing anyway